src/playercards/cards/KohakuNarukami.ttslua

```diff
@@ -0,0 +1,135 @@
+require("playercards/CardsWithHelper")
+local blessCurseManagerApi      = require("chaosbag/BlessCurseManagerApi")
+local guidReferenceApi = require("core/GUIDReferenceApi")
+local playermatApi       = require("playermat/PlayermatApi")
+local searchLib        = require("util/SearchLib")
+local tokenManager = require("core/token/TokenManager")
+local updated
+
+function updateSave()
+  self.script_state = JSON.encode({ isHelperEnabled = isHelperEnabled })
+end
+
+function onLoad(savedData)
+  self.addTag("CardWithHelper")
+  if savedData and savedData ~= "" then
+    local loadedData = JSON.decode(savedData)
+    isHelperEnabled = loadedData.isHelperEnabled
+  end
+  checkOptionPanel()
+  updateDisplay()
```

Comment:
Can remove this call since `checkOptionPanel()` will already call `updateDisplay()`